bundle/Cache/PersistenceCachePurger.php

```diff
@@ -69,7 +69,7 @@ public function all()
             return;
         }
 
-        $this->cache->clear();
+        //$this->cache->clear();
```

Comment:
@andrerom I'm going to need some help with this one, or rather, with any commented out call to cache client here...

Reply:
Indeed.

well first we need to decide if we are going to have a branch only supporting 7.0 or have two adapters here being injected depending on version.

Reply:
I'm all for keeping it simple. 2.0 legacy bridge supporting only kernel 7.0.

If needed, this adapter could be backported than to 1.x.

Reply:
Ok, I'll push commit for this one.

Reply:
Don't forget other commented out calls, those are all I didn't know what to do with :)